public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
+    body{
+background-color:#FCFCFC;
+animation-name: graytocolorimg;
+animation-duration: 4s;
```

Comment:
I was thinking of a very subtle animation; maybe to catch the user's attention that this is an important message.